src/components/structures/RoomView.js

```diff
@@ -586,7 +586,6 @@ module.exports = createReactClass({
                 return ContentMessages.sharedInstance().sendContentListToRoom(
                     [payload.file], this.state.room.roomId, MatrixClientPeg.get(),
                 );
-                break;
```

Comment:
this seems unrelated?

Reply:
It is, but its just a small delint, saw it out of the corner of my eye and couldn't help myself, its a break after a return

Reply:
oh, I didn't even notice that. It should probably drop the return and break instead, unless the return is doing something.

Reply:
oh thats an even better point, onAction is a void method iirc